### PR TITLE
Fallon responses

### PR DESCRIPTION
**Pull request by jonifallon**
ran out of time

### CODE REVIEW

**Comment by jonifallon** [comment]
I copied code from a previous day but didn't have time to test it, so I'm 100% positive it's not working.  I'm not fully comfortable with this yet....but I will get there.

File: lib/diagnostic.js [hunk 1]
```diff
@@ -7,13 +7,21 @@
 // (uppercase strings without punctuation)
 // and returns that array.
 const normalizeWords = function (/* */) {
-  /* your response here */
+  let normalizedWords = [];
+  for (let i = 0; i < words.length; i++) {
+    normalizedWords[i] = words[i].toUpperCase()
+  }
+  return normalizedWords
```
Comment (benjimelito) [inline lib/diagnostic.js:14]: Looks good here, although you simply forgot to remove punctuation. A simple way to do this is through regex.

File: lib/diagnostic.js [hunk 2]
```diff
@@ -7,13 +7,21 @@
...
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function () {
+  let uniqueWords = [];
+  for (let i = 0; i < words.length; i++) {
+    uniqueWords[i] = words[i].toUpperCase()
+  }
+  return uniqueWords
```
Comment (benjimelito) [inline lib/diagnostic.js:24]: This function looks pretty similar to your function above. What we want to do is: Check if a word has already been seen, and if it hasn't, store it in an object to check against.

File: lib/diagnostic.js [hunk 3]
```diff
@@ -22,13 +30,26 @@ const uniqueWords = function (/* */) {
 // a second argument.
 
 const wordCount = function (/* */) {
-  /* your response here */
+  const i
```
Comment (benjimelito) [inline lib/diagnostic.js:33]: You don't need to define `i` outside of the for loop, and in fact it is better to define it inside the loop using the `let` keyword

**Comment by benjimelito** [comment]
No worries on running out of time, looks like you're on the right track. 2/4

**Pull request closed without merging**